### ISSUE TITLE
Obfuscate secrets

### ISSUE THREAD

**Issue by mainanick** [ichtrojan/horus#23]
Nice request logging lib 🌶️🔥.Is it possible to obfuscate secrets like oauth tokens automatically before savings them?

**Comment by ichtrojan** [comment]
Yeah, we are currently looking into this. Fields we have in mind are password, pin and token.
